server_test.go

```diff
@@ -40,7 +40,7 @@ func TestAppHandler(t *testing.T) {
 }
 
 func TestAppHandlerMissingCookieErr(t *testing.T) {
-	t.Log("given a missing cookie, no html and a 301 response should return")
+	t.Log("given a missing cookie, no html and a 401 response should return")
```

Comment:
realized we were `301`ing and that means permanent redirect that browsers can cache. A `302.